CallAutomation_CallLiveTranscription/CallAutomationLiveTranscription/Helper.cs

```diff
@@ -0,0 +1,85 @@
+        {
+            try
+            {
+                while (webSocket.State == WebSocketState.Open || webSocket.State == WebSocketState.CloseSent)
+                {
+                    var buffer = new byte[1024 * 4];
+                    var cancellationToken = new CancellationTokenSource(TimeSpan.FromSeconds(60)).Token;
+                    WebSocketReceiveResult receiveResult = await webSocket.ReceiveAsync(new ArraySegment<byte>(buffer), cancellationToken);
+
+                    while (!receiveResult.CloseStatus.HasValue)
+                    {
+                        string msg = Encoding.UTF8.GetString(buffer, 0, receiveResult.Count);
+
+                        //var response = StreamingDataParser.Parse(buffer.Take(receiveResult.Count).ToArray());
+
+                        var response = StreamingDataParser.Parse(msg);
+
+                        if (response != null)
+                        {
+                            if (response is TranscriptionMetadata transcriptionMetadata)
+                            {
+                                Console.WriteLine("***************************************************************************************");
+                                Console.WriteLine("TRANSCRIPTION SUBSCRIPTION ID-->" + transcriptionMetadata.TranscriptionSubscriptionId);
+                                Console.WriteLine("LOCALE-->" + transcriptionMetadata.Locale);
+                                Console.WriteLine("CALL CONNECTION ID--?" + transcriptionMetadata.CallConnectionId);
+                                Console.WriteLine("CORRELATION ID-->" + transcriptionMetadata.CorrelationId);
+                                Console.WriteLine("***************************************************************************************");
+                            }
+                            if (response is TranscriptionData transcriptionData)
+                            {
+                                Console.WriteLine("***************************************************************************************");
+                                Console.WriteLine("TEXT-->" + transcriptionData.Text);
+                                Console.WriteLine("FORMAT-->" + transcriptionData.Format);
+                                Console.WriteLine("OFFSET-->" + transcriptionData.Offset.Ticks);
+                                Console.WriteLine("DURATION-->" + transcriptionData.Duration.Ticks);
+                                Console.WriteLine("PARTICIPANT-->" + transcriptionData.Participant.RawId);
+                                Console.WriteLine("CONFIDENCE-->" + transcriptionData.Confidence);
+                                Console.WriteLine("RESULT STATUS-->" + transcriptionData.ResultState);
+                                foreach (var word in transcriptionData.Words)
+                                {
+                                    Console.WriteLine("TEXT-->" + word.Text);
```

Comment:
make `word.Text` to indicate that its words object you are printing. Since we already have the same log at line 45